bindings/node.js/kzg.cxx

```diff
@@ -371,13 +334,12 @@ Napi::Value VerifyBlobKzgProofBatch(const Napi::CallbackInfo& info) {
     blobs,
     commitments,
     proofs,
-    blobs_count,
+    count,
     kzg_settings
   );
 
   if (ret != C_KZG_OK) {
     Napi::TypeError::New(env, "Error in verifyBlobKzgProofBatch").ThrowAsJavaScriptException();
-    result = env.Null();
```

Comment:
Hmm, I'm confused a bit. Why did we change `result` to default to `Undefined`? Shouldn't we always return `Null` when there is a bad verification? In the other verify functions we return `Null` but here we return `Undefined`.

Reply:
updated.